app/models/article.rb

```diff
@@ -211,6 +204,18 @@ class Article < ApplicationRecord
     boosted_additional_tags String, default: ""
   end
 
+  def self.active_help
```

Comment:
I saw you changed this to a class method from a scope. While I won't make a difference in this specific case, I want to point out that the two are not 100% the same.

If a scope returns nothing (e.g. because of a guard clause, the result will default to `.all`:

```ruby
class Article
  scope :scope_test, -> { return nil }
end

Article.scope_test.count
#=> 25
```

The same is not true for a class method:

```ruby
class Article
  def self.class_method_test; end # implicit nil return
end

Article.class_method_test.count
#=> NoMethodError: undefined method `count' for nil:NilClass
```

If you look at [the implementation of scope](https://api.rubyonrails.org/v6.1.0/classes/ActiveRecord/Scoping/Named/ClassMethods.html#method-i-scope) you'll notice that it starts from `all` and then narrows the collection down:

```ruby
scope = all._exec_scope(*args, &body)
scope = scope.extending(extension) if extension
scope
```

Reply:
Good to know!